### PR TITLE
Newfood Part 1: The Prequel

### PR DESCRIPTION
**Pull request by thgvr**
## About The Pull Request
Ports https://github.com/BeeStation/BeeStation-Hornet/pull/8748

I forgot this one

## Changelog

:cl:
RKz, Ghommie, tralezab, cacogen, Krysonism, qustinnus
add: Added Pizza and Burgers to Newfood.
delete: Temporarily removes custom burgers and pizza
tweak: processable element uses the correct grammar and references
code: split pizzaspaghetti.dmi into pizza.dmi and spaghetti.dmi
/:cl:

### CODE REVIEW

**Comment by thgvr** [comment]
todo: fix multitools. this pr is fucked up

**Comment by goober3** [comment]
hi. i want to be included in the conversation too. multitools, am i right.

File: code/datums/components/remote_materials.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@ handles linking back and forth.
 	src.allow_standalone = allow_standalone
 
 	RegisterSignal(parent, COMSIG_PARENT_ATTACKBY, PROC_REF(OnAttackBy))
-	RegisterSignal(parent, COMSIG_ATOM_MULTITOOL_ACT, PROC_REF(OnMultitool))
+	//RegisterSignal(parent, COMSIG_ATOM_MULTITOOL_ACT, PROC_REF(OnMultitool)) todo: port component rework of multitool buffers (bee)
```
Comment (Tsar-Salat) [inline code/datums/components/remote_materials.dm:27]: Buffer component came after my newfood part 1, I dont think it will be necessary.

**Comment by thgvr** [comment]
Kill me with hammers if I don't come back to this

**Pull request closed without merging**